source/Public/SQL Database/Get-FabricSQLDatabase.ps1

```diff
@@ -74,20 +78,24 @@ function Get-FabricSQLDatabase {
     )
 
     begin {
+
         Confirm-TokenState
 
-        if ($PSBoundParameters.ContainsKey("SQLDatabaseName") -and $PSBoundParameters.ContainsKey("SQLDatabaseId")) {
-            throw "Parameters SQLDatabaseName and SQLDatabaseId cannot be used together"
-        }
     }
 
     process {
+
+        if ($PSBoundParameters.ContainsKey("SQLDatabaseName") -and $PSBoundParameters.ContainsKey("SQLDatabaseId")) {
+            Write-Warning "The parameters SQLDatabaseName and SQLDatabaseId cannot be used together"
+            return
+        }
+
         if ($PSCmdlet.ParameterSetName -eq 'WorkspaceObject') {
             $WorkspaceId = $Workspace.id
         }
 
         # Create SQLDatabase API
-        $uri = "$($FabricSession.BaseApiUrl)/workspaces/$WorkspaceId/SqlDatabases"
+        $uri = "$($FabricSession.BaseApiUrl)/workspaces/$WorkspaceId/SQLDatabases"
```

Comment:
```suggestion
        $uri = "$($FabricSession.BaseApiUrl)/workspaces/$WorkspaceId/sqlDatabases"
```
lower-case for "sql" is according to documentation. No change please.
https://learn.microsoft.com/en-us/rest/api/fabric/sqldatabase/items/list-sql-databases?tabs=HTTP

Reply:
good spot, thank you